### PR TITLE
Make apps optional for the mdb-realm action

### PR DESCRIPTION
**Pull request by nirinchev**
add another output for the cluster name/credentials

These changes are intended for consumers who don't use the cli to import applications and instead use the admin API directly.

### CODE REVIEW

**Comment by nirinchev** [comment]
Supersedes https://github.com/realm/ci-actions/pull/4.

File: mdb-realm/deployApps/action.yml [hunk 1]
```diff
@@ -18,14 +18,16 @@ inputs:
...
+    required: false
     description: The path to the apps to import into MongoDB Realm.
   differentiator:
     required: true
     description: Provide a differentiator for the build - e.g. "{{ matrix.os }}-{{ matrix.node-version }}"  or "integration tests"
 outputs:
   deployedApps:
     description: "A base64-encoded json object keyed on the app name containing the app ids of the deployed apps - e.g. { \"app1\": \"app1-abcde\"}"
+  clusterName:
+    description: The name of the created cluster
```
Comment (kraenhansen) [inline mdb-realm/deployApps/action.yml:30]: Will this fail if the name is already taken?

**Review by kraenhansen — APPROVED** [review]
Looks about right. Thanks for moving this change along.

File: mdb-realm/deployApps/action.yml (hunk 1 above)
Comment (nirinchev) [reply mdb-realm/deployApps/action.yml:30]: This is an output variable and the name of the cluster is generated and should be unique. But yes - the action will fail if it tries to create a cluster that already exists (e.g. if we re-run the action without cleaning up a previous deployment).

**Pull request merged**